DEVELOPING_OPAL/demos/src/main/scala/org/opalj/ai/UselessComputations.scala

```diff
@@ -24,79 +25,72 @@ object UselessComputations extends ProjectAnalysisApplication {
-        with domain.l0.DefaultTypeLevelDoubleValues
-        with domain.l0.TypeLevelFieldAccessInstructions
-        with domain.l0.TypeLevelInvokeInstructions
-        with domain.l0.TypeLevelDynamicLoads
-        with domain.l1.DefaultReferenceValuesBinding
-        with domain.l1.DefaultIntegerRangeValues
-        with domain.l1.DefaultLongValues
-        with domain.l1.ConcretePrimitiveValuesConversions
-        with domain.l1.LongValuesShiftOperators {
+            with domain.DefaultSpecialDomainValuesBinding
+            with domain.TheProject
+            with domain.TheMethod
+            with domain.DefaultHandlingOfMethodResults
+            with domain.IgnoreSynchronization
+            with domain.ThrowAllPotentialExceptionsConfiguration
+            with domain.l0.DefaultTypeLevelFloatValues
+            with domain.l0.DefaultTypeLevelDoubleValues
+            with domain.l0.TypeLevelFieldAccessInstructions
+            with domain.l0.TypeLevelInvokeInstructions
+            with domain.l0.TypeLevelDynamicLoads
+            with domain.l1.DefaultReferenceValuesBinding
+            with domain.l1.DefaultIntegerRangeValues
+            with domain.l1.DefaultLongValues
+            with domain.l1.ConcretePrimitiveValuesConversions
+            with domain.l1.LongValuesShiftOperators {
 
         override def maxCardinalityOfIntegerRanges: Long = 4L
     }
 
     override def title: String = "useless computations"
 
-    override def description: String = {
-        "identifies computations that are useless, e.g., "+
-            "comparison against null if the value is known not be null"
-    }
+    override def description: String = "identifies computations that are useless, e.g., " +
+        "comparison against null if the value is known not be null"
 
     override def doAnalyze(
         theProject:    Project[URL],
         parameters:    Seq[String],
-        isInterrupted: () => Boolean
-    ): BasicReport = {
+        isInterrupted: () => Boolean): BasicReport = {
 
         val results = {
             val results = for {
-                classFile <- theProject.allProjectClassFiles.par
+                classFile                     <- theProject.allProjectClassFiles.par
                 method @ MethodWithBody(body) <- classFile.methods
-                result = BaseAI(method, new AnalysisDomain(theProject, method))
+                result                         = BaseAI(method, new AnalysisDomain(theProject, method))
             } yield {
                 import result._
                 val results = collectPCWithOperands(domain)(body, operandsArray) {
                     case (
-                        pc,
-                        IFNULL(_) | IFNONNULL(_),
-                        Seq(domain.DomainReferenceValueTag(value), _*)
-                        ) if value.isNull.isYesOrNo =>
-                        UselessComputation(method, pc, "useless comparison with null")
+                            pc,
```

Comment:
Why is this indented so much?

Reply:
I guess this affected by `ctorSite = 8 # Double Indent` which I just copied over.